src/ResourceManager/Sql/Commands.Sql/help/AzureRM.Sql.md

```diff
@@ -62,6 +62,9 @@ Gets the details of an import or export of an Azure SQL Database.
 ### [Get-AzureRmSqlDatabaseIndexRecommendations](Get-AzureRmSqlDatabaseIndexRecommendations.md)
 Gets the recommended index operations for a server or database.
 
+### [Get-AzureRmSqlDatabaseInstanceFailoverGroup](Get-AzureRmSqlDatabaseInstanceFailoverGroup.md)
+{{Fill in the Synopsis}}
```

Comment:
@jugeorge please make sure to fill in these sections that say `{{Fill in the Synopsis}}`